docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+
+Identity in Hyperledger Fabric:<br>
+https://hyperledger-fabric.readthedocs.io/en/latest/identity/identity.html#identity
+
+MSPs in Hyperledger Fabric:<br>
+https://hyperledger-fabric.readthedocs.io/en/latest/membership/membership.html#membership-service-provider-msp
+
+Registration and Enrollment:<br>
+https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#overview-of-registration-and-enrollment
+
+Registering an Identity:<br>
+https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#register-an-identity
+
+Enrolling an Identity:<br>
+https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#enroll-an-identity
+
+
+It is key to understand that an identity contains the public certificate and private key used for encryption.  The public certificate is a signed certificate from the CA and is referred to generically as a certificate in the following table.
+
+The table shows the types of certificates used by Hyperledger Fabric.
+
+
+## Technical details
+
+**Note:** Certificates for Organization CAs and the TLS CA are being listed separately.  This is due to the differing purposes for the CAs.   The Organization CA supplies signing certificates and the TLS CA supplies TLS certificates for securing connections.
+
+### Organization CA Certificates
+
+| Certificate | Description | Location | Impact if expired | Reference |
+|----------------|-----|---------------------------|-----------|---|
+| Root Certificate | Controlling certificate used to sign all certificates issued by the CA. | Stored on disk in the CA directory.<br><br>ca-cert.pem | CA root certificates are issued for 15 years. A New certificate must be issued.  |  |
+| CA Admin Certificate | Certificate used to make administrative requests to the CA, such as to register new users. | Depends on implementation <br><br><code>msp <br />&nbsp;&nbsp;&nbsp;├── IssuerPublicKey <br>&nbsp;&nbsp;&nbsp;├── IssuerRevocationPublicKey <br>&nbsp;&nbsp;&nbsp;├── cacerts <br>&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── localhost-7053.pem <br>&nbsp;&nbsp;&nbsp;├── keystore <br>&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── key.pem <br>&nbsp;&nbsp;&nbsp;├── signcerts <br>&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── cert.pem <br>&nbsp;&nbsp;&nbsp;└── user <br></code> | Cannot register or enroll new identities, but transaction traffic does not stop.| https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer-org-s-ca-admin |
+
+### TLS CA Certificates
+
+| Certificate | Description | Location | Impact if expired | Reference |
+|----------------|-----|---------------------------|-----------|---|
+| Root Certificate | Certificate used to make administrative requests to the CA, such as to register new users. | Stored on disk in the CA directory. <br><br>ca-cert.pem | CA root certificates are issued for 15 years. A New certificate must be issued.  |  |
```

Comment:
```
Certificate used to make administrative requests to the CA, such as to register new users
```
I'm not 100% sure that's correct. @yacovm could you pls weigh in?

Reply:
> ```
> Certificate used to make administrative requests to the CA, such as to register new users
> ```
> 
> I'm not 100% sure that's correct. @yacovm could you pls weigh in?

I honestly don't understand why we couple management (registering users, etc.) of a specific CA implementation (Fabric CA) with the role of certificates in Fabric.

What if I want to use a commercial CA for TLS? What if I have my own CA for my company and I don't use Fabric-CA so the terminology of registering users does not apply to me?

Reply:
We are trying to document what the types of certs are used for and where they are located. Not to cover every scenario in this iteration.